public/configurations/visualizations/aar-enterprise-app-pie.json

```diff
@@ -0,0 +1,36 @@
+{
+    "id": "aar-enterprise-app-pie",
+    "graph": "PieGraph",
+    "title": "All Enterprise Applications",
+    "author": "Bharat Mukheja",
+    "creationDate": "09/21/2017",
+    "data": {
+        "sliceColumn": "Sum of MB",
+        "labelColumn": "L7Classification",
+        "otherOptions": {
+            "label": "Others",
+            "limit": 10
+        },
+        "tooltip": [
+            { "column": "L7Classification", "label": "L7 Classification" },
+            { "column": "Sum of MB", "format": "s"}
+        ],
+        "percentages": true,
+        "percentagesFormat": ",.1%",
+        "pieLabelRadius": 0.55,
+        "pieOuterRadius": 0.95,
+        "legend": {
+            "show": true,
+            "orientation": "horizontal",
+            "circleSize": 1,
+            "labelOffset": 2
+        }
+    },
+    "listeners": [{
+      "redirect": "/dashboards/dateHistogramExample",
+      "params": {
+          "app": "L7Classification"
+      }
+    }],
+    "query": "aar-enterprise-app-vertical-bar"
```

Comment:
Where is the change in the dashboard "aarEnterprise" where the current visualization is replaced by this one?